src/main/java/com/epam/training/auction_backend/entity/User.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.training.auction_backend.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "USERS")
+public class User {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "ID", unique = true, nullable = false)
+	private int id;
+
+	@Column(name = "USERNAME", unique = true)
+	private String userName;
+
+	@Column(name = "PASSWORD", unique = true)
+	private String password;
+
+	public User() {
```

Comment:
you do not have to define the default constructor if you do not want to change its implementation or visibility